rest/route/user.go

```diff
@@ -300,22 +299,13 @@ func (h *allUsersPermissionsGetHandler) Factory() gimlet.RouteHandler {
 func (h *allUsersPermissionsGetHandler) Parse(ctx context.Context, r *http.Request) error {
 	err := utility.ReadJSON(r.Body, &h.input)
 	if err != nil {
-		return gimlet.ErrorResponse{
-			StatusCode: http.StatusBadRequest,
-			Message:    fmt.Sprintf("error parsing request body: %s", err.Error()),
-		}
+		return errors.Wrap(err, "reading permissions request from JSON request body")
 	}
 	if !utility.StringSliceContains(evergreen.ValidResourceTypes, h.input.ResourceType) {
-		return gimlet.ErrorResponse{
-			StatusCode: http.StatusBadRequest,
-			Message:    fmt.Sprintf("resource_type must be in %v", evergreen.ValidResourceTypes),
-		}
+		return errors.Errorf("invalid resource type '%s'", h.input.ResourceType)
 	}
 	if h.input.ResourceId == "" {
-		return gimlet.ErrorResponse{
-			StatusCode: http.StatusBadRequest,
-			Message:    "resource_id is required",
-		}
+		return errors.New("resource ID is required")
```

Comment:
`must specify resource ID`

Reply:
Same thing as above about not being overly nitpicky about base error messages.